samples/full/0183-RMC-export.json

```diff
@@ -1,14 +1,15 @@
-{
+{ "self":"urn:mrn:imo:mmsi:366982330",
   "vessels": {
     "urn:mrn:imo:mmsi:366982330": {
       "mmsi": "230099999",
```

Comment:
Discrepancy, could you please fix it at the same time, even this was already there?